### DISCUSSION TITLE
BlueR - BlueZ version compatibility

### DISCUSSION THREAD

**Discussion by thives** [bluez/bluer#71] [Q&A] [answered]
Is there a way to determine version compatibility between different versions of BlueR and BlueZ?
An example: 
BlueZ added negotiated mtu to the dbus interface of ReadValue and WriteValue in version 5.51. 
All versions of BlueR will in this case fail for sure when relevant communication is happening with BlueZ version 5.50.

Maybe not the best example since it is stated that BlueR has been tested with BlueZ 5.60, and may or may not work with older versions. But there could be other incompatibilities current and future, and it would be very helpful to have a matrix of verified/tested version combinations.

**Comment by surban** [comment, marked as answer]
I agree!

A prerequisite for this would be automated testing of BlueR, which we currently don't have. As long as we can't build that matrix automatically I'm afraid it would get outdated very quickly and become useless.
